README.md

```diff
@@ -102,3 +102,19 @@ Then, from within your project folder:
 npm run build
 surge public my-project.surge.sh
 ```
+
+### With [Netlify](https://www.netlify.com/) CLI
+
+Install `netlify-cli` if you haven't already:
+
+```bash
+npm install -g netlify-cli
+```
+
+Then, from within your project folder:
+
+```bash
+netlify deploy --prod
+```
+
+Follow the instructions and be sure to set `public` as Publish Directory.
```

Comment:
I love this callout for making sure to select the `public` directory because I totally didn't catch it myself when I tried to deploy so I know this is helpful!

It looks like the CLI added a flag to use instead too, so we can update the command to:

```bash
netlify deploy --dir=public --prod
```

and leave the rest to "Follow the rest of the setup instructions"